k8s/namespaces/reform-scan/blob-router-service/aat.yaml

```
@@ -15,22 +13,9 @@ metadata:
     repository.fluxcd.io/functional: java.functionaltests.image
```

@adusumillipraveen don't these need updating to have the java prefix?

```suggestion
    repository.fluxcd.io/java.functional: java.functionaltests.image
```

---

yes, they need to.

---

do these annotations need to be in the aat.yaml file as well?

---

If they are not in all envrionments.  In our case(draft-store), we had smoke tests enabled only on aat.

---

also isn't it `java.functionaltests`/`java.smoketests`

---

yes